tests/integration/targets/script/tasks/main.yml

```diff
@@ -74,15 +74,15 @@
   ansible.builtin.set_fact:
     files_to_check:
```

Comment:
there's something fishy - the files don't seem to exist? At least the ansible-test is failing, not seeing them.

Reply:
right. as i said it passes locally and the files are generated fine . in fact in those logs it finds the other file OK '/tmp/dummy/test/test-file.txt'

but not the /tmp/artifacts/ ones. the tasks are completing though e.g.

```
TASK [script : Run simple no failing script] ***********************************
[0;33mchanged: [testhost][0m
```

but then it cannot find the file that produced 

```
[0;31mfailed: [testhost] (item={'changed': False, 'stat': {'exists': False}, 'invocation': {'module_args': {'path': '/tmp/artifacts/ci_script_000_run_simple_no_failing_script.sh', 'follow': False, 'get_md5': False, 'get_checksum': True, 'get_mime': True, 'get_attributes': True, 'checksum_algorithm': 'sha1'}}, 'failed': False, 'item': {'key': '/tmp/artifacts/ci_script_000_run_simple_no_failing_script.sh', 'value': 'cf984cf63c514cb4ff53aad1e862a3453298644b'}, 'ansible_loop_var': 'item'}) => {[0m
```

Reply:
```
TASK [script : Create needed directories] **************************************
[0;32mok: [testhost] => (item=artifacts)[0m
[0;32mok: [testhost] => (item=logs)[0m
```

Reply:
ok problem is that they are now (both ci_make and ci_script) writing to ci_script_foo in the /tmp logs ... so i had to change the expected file attributes a bit in the last push. 

the ansible-test is now green there https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openstack-k8s-operators_ci-framework/653/pull-ci-openstack-k8s-operators-ci-framework-main-ansible-test/1747914305224839168